tests/test_metrics.py

```diff
@@ -238,6 +365,7 @@ def test_evaluate_in_batches(
     metrics,
     batch_size,
 ):
+    model.train_loader.negative_sample_type = 'item'
```

Comment:
Should we test `test_evaluate_in_batches` for user-based negative sampling as well?